documents/policies/Competition Requirements.md

```diff
@@ -0,0 +1,141 @@
+# WCA Competition Requirements Policy
+<div class="version">
+### Version 1.0
+</div>
+
+## Purpose
+The purpose of this policy is to define the requirements and processes that a competition must meet and follow in order to be recognized as a WCA Competition.
+
+## Policy
+### WCA Competitions
+An official WCA Competition must:
+
+1. Be announced at least 28 days (four weeks) before 00:00 UTC of the start date of the competition on the WCA website.
+   1. WCA Delegates should allow a suitable time frame for competitions to be announced after they are confirmed to the WCA Competition Announcement Team.
+2. Comply with the [WCA Regulations](wca{regulations}).
+3. Comply with all [WCA Policies](wca{documents}).
+4. Be publicly accessible.
+5. Be open to all competitors and spectators who wish to attend, up to the maximum capacity stated on the website or otherwise permitted by the venue.
+6. Be held on a single date or on two or more consecutive days.
+7. Have at least one official event.
+8. Have at least one competitor.
+9. Have at least one designated WCA Delegate in attendance.
+   1. The WCA Delegate for a competition is responsible for:
+      1. Submitting the Delegate Report to the WCA Regulations Committee and the WCA Quality Assurance Committee regarding adherence to WCA Regulations and quality standards during the competition, the overall course of the competition, and any incidents that happened. The report must be submitted within one week of the end of the competition.
+      2. Submitting the competition results to the WCA Results Team within one week of the end of the competition.
+         1. Results should be double-checked and free of errors before submission.
+         2. All scramble sequences used during a competition must be sent with the results of the competition.
+            1. All scramble sequences must be labeled with the events, rounds, and groups for which they were used. If there are multiple groups on a round, it is not necessary to identify which competitor was in which group.
+         3. Physical records (e.g. score sheets, written solutions from 3x3x3 Fewest Moves) must be retained for a minimum of 90 days. Digital records (e.g. scramble sequences, results) must be retained for 7 years.
```

Comment:
Aren't scrambles stored in the WCA database?
Do delegates have to keep the scrambles/results file in their computer for 7 years?

Reply:
Yes. What if the wca database is deleted?
Btw it doesn’t have to be on your local computer. This can also be possible via maintaining the results email thread (ie just don’t delete it)